azure/template.json

```diff
@@ -128,7 +128,8 @@
         "instanceName": "[if(equals(toUpper(parameters('environmentName')),'PROD'),'',parameters('environmentName'))]",
         "databaseName": "[concat(variables('resourceNamePrefix'), '-db')]",
         "serviceBusName": "[concat(variables('resourceNamePrefix'), '-ns')]",
-        "configNames": "SFA.DAS.AssessorService.Web,SFA.DAS.Assessor.GovSignIn"
+        "configNames": "SFA.DAS.AssessorService.Web,SFA.DAS.Assessor.GovSignIn",
+        "ApplicationIdentifierUri":"[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', '-assextapi-as-ar'))]"
```

Comment:
This would be more consistent with the other variables:
"externalApiApplicationIdentifierUri": "[concat(variables('resourceNamePrefix'), 'extapi-as-ar')]",